arbutus/block-template-parts/footer.html

```diff
@@ -9,11 +9,7 @@
 <!-- /wp:paragraph -->
 
 <!-- wp:social-links {"iconColor":"primary","iconColorValue":"#343e47","size":"has-small-icon-size","className":"is-style-logos-only"} -->
-<ul class="wp-block-social-links has-small-icon-size has-icon-color is-style-logos-only"><!-- wp:social-link {"url":"#","service":"twitter"} /-->
-
-<!-- wp:social-link {"url":"#","service":"instagram"} /-->
-
-<!-- wp:social-link {"url":"#","service":"tumblr"} /--></ul>
+<ul class="wp-block-social-links has-small-icon-size has-icon-color is-style-logos-only"></ul>
```

Comment:
You might want to make this a navigation block that implements a social nav. You can do that like this:

```<!-- wp:navigation {"itemsJustification":"center","overlayMenu":"never","__unstableSocialLinks":"social"} /-->```

Reply:
Hmm... I do want a specific look for the social links nav in the footer (icons only, no bg, custom color, custom icon size), and I can't seem to achieve this inside a nav menu. Am I missing something?
![image](https://user-images.githubusercontent.com/4550351/142302089-cbad8715-1b49-4d4b-a067-9a9a9e4e03a8.png)

Reply:
We need [this PR](https://github.com/Automattic/themes/pull/4984) to be merged first...

Reply:
that PR will only help with the color, not the custom size, I believe

Reply:
we could do custom size with CSS for now..

Reply:
I used your simplified setup, and I think it's good to leave like that.